Explorer/Assets/DCL/SceneLoadingScreens/SceneLoadingScreenView.cs

```diff
@@ -124,5 +124,23 @@ public async UniTask HideTipWithFadeAsync(int index, float duration, Cancellatio
 
             tips[index].gameObject.SetActive(false);
         }
+
+        public async UniTask ChangeBackgroundColor(Material material, string property, Color toColor, float duration, CancellationToken cancellationToken = default)
```

Comment:
This method should simplify its parameters:
```
public async UniTask SetBackgroundColor(Color toColor, float duration, CancellationToken cancellationToken = default)
```

Reply:
changed